autoload/fzf/vim.vim

```diff
@@ -1192,11 +1194,12 @@ function! s:commits(range, buffer_local, args)
     return s:warn('Not in git repository')
   endif
 
-  let source = 'git log '.get(g:, 'fzf_commits_log_options', '--color=always '.fzf#shellescape('--format=%C(auto)%h%d %s %C(green)%cr'))
-  let current = expand('%')
+  let git_root_escaped = fzf#shellescape(s:git_root)
+  let source = 'git -C ' . git_root_escaped . ' log '.get(g:, 'fzf_commits_log_options', '--color=always '.fzf#shellescape('--format=%C(auto)%h%d %s %C(green)%cr'))
```

Comment:
I mean, we're repeating `'git -C ' . root_escaped . ' ` several times. So my suggestion was to put the whole thing in a variable.

```vim
let prefix = 'git -C ' . fzf#shellescape(root) . ' '
let source = prefix . 'log ' ...
```

Reply:
Yup that certainly looks cleaner. Fixed by 098f907 and rebased.